openapi_python_client/schema/openapi_schema_pydantic/schema.py

```diff
@@ -406,7 +406,7 @@ class Schema(BaseModel):
     Other than the JSON Schema subset fields, the following fields MAY be used for further schema documentation:
     """
 
-    nullable: bool = False
+    nullable: Optional[bool] = None
```

Comment:
Again, why not default to `False` rather than needing to do extra conversion after the fact? I don't think it being unset is any different semantically to us than it being `False`.

Reply:
Agreed, resolved in https://github.com/triaxtec/openapi-python-client/pull/426/commits/9dc2cacc267ad317a6af584ab815fb1b6f42740f